### DISCUSSION TITLE
Coomer and Bunker Scrape failure

### DISCUSSION THREAD

**Discussion by DagonSIV** [jbsparrow/CyberDropDownloader#231] [Q&A] [answered]
Just keep on getting scrape failure with whatever link i put in and i have no clue what's going wrong

**Comment by DagonSIV** [comment]
This is the error from the log  [WinError 1225]

**Reply by jbsparrow** [reply]
Very indescript. Would appreciate your full log. I think it's an issue with what your `scan_folder` is set to, can't be sure without the full log though.

**Reply by DagonSIV** [reply]
I've found the issue it's my VPN when it's off it works fine but when it's on it fails

**Comment by jbsparrow** [comment, marked as answer]
There is nothing we can do to make your VPN work with the websites in question. They are blocking access from you VPN.
